pr_agent/settings/configuration.toml

```diff
@@ -1,6 +1,6 @@
 [config]
 model="gpt-4"
-fallback-models=["gpt-3.5-turbo-16k", "gpt-3.5-turbo"]
+fallback_models=["gpt-3.5-turbo-16k"]
```

Comment:
this was inserted here to support accounts without access to gpt-4 or gpt-3.5-turbo-16k

Reply:
@okotek gpt-3 won't be able to handle most PRs, simply because the context length is too small (4K)
I don't think it can give a reasonable usage experience for PR-Agnet